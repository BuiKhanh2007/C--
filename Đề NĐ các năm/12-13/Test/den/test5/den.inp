82 140
145 218
100 295
179 345
17 173
8 247
87 283
20 251
199 266
