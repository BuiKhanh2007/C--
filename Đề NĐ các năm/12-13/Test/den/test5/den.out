674
937
2594
2416
1675
2748
2544
2725
988
